artifact-version-diff/src/main/java/io/quarkus/qe/PrepareOperation.java

```diff
@@ -38,7 +40,8 @@ public static Path prepareVersionPluginOutput() throws IOException {
         Path tmpDirectory = Files.createDirectories(Paths.get(System.getProperty("java.io.tmpdir"), generatedRandomDirName));
 
         LOG.info("Cloning Quarkus repository");
-        String branch = Objects.requireNonNull(System.getProperty("quarkus.repo.tag"), "The quarkus.repo.tag property wasn't set.");
+        String branch = normalizeVersionForUpstream(Objects.requireNonNull(System.getProperty("quarkus.repo.tag"), "The quarkus.repo.tag property wasn't set."));
```

Comment:
why `normalizeVersionForUpstream` method? Isn't just bad `quarkus.repo.tag` passed to the execution?

Reply:
I would say that the line in a job that is trying to figure out quarkus tag needs to be updated to handle case when "temporary" versions are used